mkdocs.yml

```diff
@@ -70,9 +71,21 @@ plugins:
       nav_file: SUMMARY.md
   - section-index
   - mkdocstrings
+  - mknotebooks
```

Comment:
Are we planning to add notebooks atm? If not I think we're safe to remove this plugin for the moment

Reply:
Removed